subsys/bluetooth/mesh/gen_ponoff_srv.c

```diff
@@ -251,7 +257,8 @@ static int bt_mesh_ponoff_srv_settings_set(struct bt_mesh_model *model,
 
 	set_on_power_up(srv, NULL, (enum bt_mesh_on_power_up)data.on_power_up);
 
-	struct bt_mesh_onoff_set onoff_set = { 0 };
+	struct bt_mesh_model_transition transition = {};
```

Comment:
Missed 0 in brackets? if no why is it needed?

Reply:
It is assumed that the pointer to the transition parameters in the set struct is never NULL.